spokestack/nsx/webrtc.py

```diff
@@ -0,0 +1,57 @@
+"""
+This module contains the class for webrtc automatic noise suppression
+"""
+import numpy as np  # type: ignore
+
+from spokestack.context import SpeechContext
+from spokestack.extensions.webrtc.nsx import WebRtcNsx  # type: ignore
+
+POLICY_MILD = 0
+POLICY_MEDIUM = 1
+POLICY_AGGRESSIVE = 2
+POLICY_VERY_AGGRESSIVE = 3
+
+
+class AutomaticNoiseSuppression:
+    """WebRTC Automatic Noise Suppression
+
+    Args:
+        sample_rate (int): audio sample rate. (Hz)
+        policy (int): level of noise supression.
```

Comment:
policy could probably use some better documentation here
see what the sphinx docs look like and consider how useful they would be to a new user